core/wiki/macros/sidebar-resizer.tid

```diff
@@ -0,0 +1,551 @@
+				matchSelector=".tc-slider"
+				$pointerup={{{ [<get.resizer.state>is[tiddler]then<sidebar-resizer-pointercancel-actions>] }}}>
+
+				<$eventcatcher
+					tag="div"
+					selector=".tc-slider-pointermove"
+					matchSelector=".tc-slider-pointermove"
+					class=<<get.active.class tc-slider-pointermove-eventcatcher>>
+					$pointerup=<<sidebar-resizer-pointercancel-actions>>
+					$pointerleave=<<sidebar-resizer-pointercancel-actions>>
+					$pointerout=<<sidebar-resizer-pointercancel-actions>>
+					$pointercancel=<<sidebar-resizer-pointercancel-actions>>
+					$pointerdown=<<sidebar-resizer-pointercancel-actions>>
+					$touchstart=<<sidebar-resizer-pointercancel-actions>>
+					$pointermove=<<sidebar-resizer-pointermove-actions>>
+					$contextmenu=<<sidebar-resizer-pointercancel-actions>>>
+
+					<div class=<<get.active.class tc-slider-pointermove>>/>
+
+				</$eventcatcher>
+
+				<%if [<templateLeft>is[variable]] [<templateLeft>is[tiddler]] %>
+
+					<div
+						style.display="flex"
+						style.position="relative"
+						style.flex-direction="column"
+						style.top="0"
+						style.left="0"
+						style.flex-basis=<<get.template-left.style.flex-basis>>
+						style.width=<<get.template-left.style.flex-basis>>
+						style.z-index=<<zIndexLeft>>
+					>
+
+						<%if [<templateLeft>is[variable]] %>
+
+							<$transclude $variable=<<templateLeft>> mode=<<mode>>/>
+
+						<% else %>
+
+							<$transclude $tiddler=<<templateLeft>> mode=<<mode>>/>
+
+						<% endif %>
+
+					</div>
+
+				<% endif %>
+
+				<%if [<get.slider.condition>match[yes]] %>
+
+					<div
+						class=<<get.resizer.class>>
+						style.position=<<get.resizer.style.position>>
+						style.left=<<get.resizer.style.left>>
+						style.width=<<get.resizer.style.width>>
+					/>
+
+				<% endif %>
+
+				<%if [<templateRight>is[variable]then<sliderCondition>!match[no]] [<templateRight>is[tiddler]then<sliderCondition>!match[no]] %>
+
+					<div
+						style.display="flex"
+						style.position="relative"
+						style.flex-direction="column"
+						style.top="0"
+						style.left=<<get.template-right.style.left>>
+						style.flex-basis=<<get.template-right.style.flex-basis>>
+						style.width=<<get.template-right.style.flex-basis>>
+					>
+
+						<%if [<templateRight>is[variable]] %>
+
+							<$transclude $variable=<<templateRight>> mode=<<mode>>/>
+
+						<% else %>
+
+							<$transclude $tiddler=<<templateRight>> mode=<<mode>>/>
+
+						<% endif %>
+
+					</div>
+
+				<% endif %>
+
+			</$eventcatcher>
+
+		<% endif %>
+
+	</$eventcatcher>
+
+</$set>
+
+\end
+
+\procedure editor-resizer(template:"",mode:"block",sliderCondition:"yes",eventCatcherClass:"")
+
+<$eventcatcher
+	tag="div"
+	class=<<get.active.class tc-slider-pointerdown-eventcatcher>>
+	selector=".tc-slider"
+	matchSelector=".tc-slider"
+	$pointerdown=<<sidebar-resizer-pointerdown-actions-inner>>
+	$pointerup=<<sidebar-resizer-pointercancel-actions>>>
+
+	<%if [<resizer.state>!match[no]] %>
```

Comment:
redundant newlines? -- I did skip this comment in all other files where there happens to be redundant new-lines